### PR TITLE
[17.0][MIG] `l10n_fr_ecotaxe`: Migration to 17.0

### PR DESCRIPTION
**Pull request by yankinmax**
Depends on:
Migration to v16
- [x] https://github.com/OCA/l10n-france/pull/448

### CODE REVIEW

File: l10n_fr_ecotaxe/models/ecotaxe_line_product.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 
 class EcotaxeLineProduct(models.Model):
-    """class for objects which can be used to save mutili ecotaxe calssification  by product."""
+    """class for objects which can be used to save
+    mutili ecotaxe calssification by product."""
```
Comment (Camille0907) [inline l10n_fr_ecotaxe/models/ecotaxe_line_product.py]:
```suggestion
    multi ecotaxe classification by product."""
```

File: l10n_fr_ecotaxe/models/ecotaxe_line_product.py [hunk 2]
```diff
@@ -28,18 +29,15 @@ class EcotaxeLineProduct(models.Model):
     amount = fields.Float(
         digits="Ecotaxe",
         compute="_compute_ecotaxe",
-        help="Ecotaxe Amount computed form Classification or forced ecotaxe amount",
+        help="Ecotaxe Amount computed form Classification or " "forced ecotaxe amount",
```
Comment (Camille0907) [inline l10n_fr_ecotaxe/models/ecotaxe_line_product.py:32]:
```suggestion
        help="Ecotaxe Amount computed form Classification or forced ecotaxe amount",
```
or
```suggestion
        help="Ecotaxe Amount computed form Classification or " 
        "forced ecotaxe amount",
```
no ?

File: l10n_fr_ecotaxe/models/ecotaxe_line_product.py [hunk 3]
```diff
@@ -28,18 +29,15 @@ class EcotaxeLineProduct(models.Model):
...
     display_name = fields.Char(compute="_compute_display_name")
 
     @api.depends("classification_id", "amount")
     def _compute_display_name(self):
         for rec in self:
-            rec.display_name = "%s (%s)" % (
-                rec.classification_id.name,
-                rec.amount,
-            )
+            rec.display_name = f"{rec.classification_id.name} {rec.amount}"
```
Comment (Camille0907) [inline l10n_fr_ecotaxe/models/ecotaxe_line_product.py:40]: Is it wanted to remove the `()` ?
I would have done 
```f"{rec.classification_id.name} ({rec.amount})"```

**Review by Camille0907 — APPROVED** [review]
Migration LGTM, only non-blocking remarks :)

File: l10n_fr_ecotaxe/models/ecotaxe_line_product.py (hunk 3 above)
Comment (yankinmax) [reply l10n_fr_ecotaxe/models/ecotaxe_line_product.py:40]: We have removed the `()` to dismiss `%s` string formatting as it's an `pre-commit` error now. There was no `()` originally behind the `amount`

File: l10n_fr_ecotaxe/models/ecotaxe_line_product.py (hunk 2 above)
Comment (yankinmax) [reply l10n_fr_ecotaxe/models/ecotaxe_line_product.py:32]: this fix is done by `pre-commit`

**Review by vvrossem — APPROVED** [review]
LGTM, thank you

**Comment by bealdav** [comment]
You problably need to revert in your project if it's possible because of

https://github.com/OCA/account-fiscal-rule/pulls/mourad-ehm

cc @yankinmax

**Pull request merged**